Library/Homebrew/test/cmd/completions_spec.rb

```diff
@@ -0,0 +1,28 @@
+# typed: false
+# frozen_string_literal: true
+
+require "cmd/shared_examples/args_parse"
+
+describe "Homebrew.completions_args" do
+  it_behaves_like "parseable arguments"
+end
+
+describe "brew completions", :integration_test do
```

Comment:
Multiple integration tests are very slow. This should be limited to a single integration test on macOS and multiple if really needed (which I'm not convinced they are given unit tests) on Linux.

Reply:
Yeah, no need for both here.